src/Playroom/Snippets/Snippets.tsx

```diff
@@ -173,22 +131,27 @@ export default ({ snippets, onHighlight, onClose }: Props) => {
           return (
             <li
               ref={isHighlighted ? highlightedEl : undefined}
+              id={`${snippet.group}_${snippet.name}_${index}`}
               key={`${snippet.group}_${snippet.name}_${index}`}
-              className={classnames(styles.snippet, {
-                [styles.highlight]: isHighlighted,
-              })}
+              className={styles.snippetPadding}
               onMouseMove={
                 isHighlighted ? undefined : () => setHighlightedIndex(index)
               }
               onMouseDown={() => closeHandler(filteredSnippets[index])}
               title={getLabel(snippet)}
             >
-              <span style={{ display: 'block', position: 'relative' }}>
-                <Text size="large">
-                  <Strong>{snippet.group}</Strong>
-                  <span className={styles.snippetName}>{snippet.name}</span>
-                </Text>
-              </span>
+              <div
+                className={classnames(styles.snippet, {
+                  [styles.highlight]: isHighlighted,
+                })}
+              >
+                <span style={{ display: 'block', position: 'relative' }}>
```

Comment:
Wonder if we need both elements here. Is there anyway to flatten this down the `li` only? Or at least combine this `div` and `span`